examples/app-demo/restaurant-backend/src/main.rs

```diff
@@ -0,0 +1,47 @@
+use anyhow::Result;
+
+use slight_http_handler_macro::register_handler;
+
+wit_bindgen_rust::import!("../../../wit/http.wit");
+wit_error_rs::impl_error!(http::Error);
+
+wit_bindgen_rust::import!("../../../wit/mq.wit");
+wit_error_rs::impl_error!(mq::Error);
+
+fn main() -> Result<()> {
+    let router = http::Router::new()?;
+    let router_with_route = router
+        .post("/make_order", "handle_make_order")?
```

Comment:
Perhaps, we can make these routes a bit more restful w/ a post to the orders resource.
```suggestion
        .post("/orders/", "handle_make_order")?
```